ndt7/model/archivaldata.go

```diff
@@ -0,0 +1,15 @@
+package model
+
+import (
+	"time"
+)
+
+// ArchivalData saves all instantaneous measurements over the lifetime of a test.
+type ArchivalData struct {
+	UUID               string
+	StartTime          time.Time
+	EndTime            time.Time
+	ServerMeasurements []Measurement
+	ClientMeasurements []Measurement
+	ClientMetadata     map[string]string `json:",omitempty" bigquery:"-"`
```

Comment:
This may contain useful information like the client name and version. If I understand correctly the `-` here you're saying you don't want to export this information to BigQuery. Can you explain/document why?